pkg/lockfile/parse-yarn-lock-v1_test.go

```diff
@@ -226,6 +226,31 @@ func TestParseYarnLock_v1_VersionsWithBuildString(t *testing.T) {
 	})
 }
 
+func TestParseYarnLock_v1_WithBuildString(t *testing.T) {
+	t.Parallel()
+
+	packages, err := lockfile.ParseYarnLock("fixtures/yarn/with-build-string.v1.lock")
+
+	if err != nil {
+		t.Errorf("Got unexpected error: %v", err)
+	}
+
+	expectPackages(t, packages, []lockfile.PackageDetails{
+		{
+			Name:      "domino",
+			Version:   "2.1.6+git",
```

Comment:
Should the +git part be trimmed/removed? Or am I misinterpreting this comment: https://github.com/google/osv-scanner/issues/574#issuecomment-1751250600

Reply:
No it should be retained as its part of the version - its `semantic` that should account for it not being relevant when comparing two versions (which it does).

Reply:
That makes sense, thanks!